test/plugins/auth.plugin.test.ts

```
@@ -1,16 +1,17 @@
+import type { UserWithAnonymous, AnonymousSession } from 'better-auth/plugins'
 import { afterAll, afterEach, describe, expect, it, mock } from 'bun:test'
-import Elysia from 'elysia'
+import { Elysia } from 'elysia'
 import { db } from '~/plugins/db.plugin'
 import { accounts, sessions, users } from '~/modules/auth'
 import { auth, authPlugin } from '~/plugins/auth.plugin'
 
 describe('Auth Plugin', () => {
-  const handler = mock(() => 'Auth')
+  const baseUrl = 'http://localhost'
+  const handler = mock(({}) => 'Auth')
```

Biome (2.3.13)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)